BitmarkRegistry/AppDelegate.swift

```diff
@@ -107,6 +107,24 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     registerAPNSSubject.onCompleted()
   }
 
+  func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:]) -> Bool {
+    guard let _ = url.scheme, let host = url.host else { return true }
+
+    if host == "authorization" {
```

Comment:
We should do switch case here for further url resource.
Moreover, should return `false` in case of not successful handling the request.